core/src/main/java/org/kohsuke/stapler/export/DataWriter.java

```diff
@@ -69,6 +70,8 @@
     void startObject() throws IOException;
     void endObject() throws IOException;
 
+    @Nonnull ExportConfig getExportConfig();
```

Comment:
Thats right. running this [query](https://github.com/search?utf8=✓&q=org%3Ajenkinsci+org.kohsuke.stapler.export.DataWriter+in%3Afile+&type=Code&ref=searchresults) on GitHub tells no external implemention - yet.